src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.GenerateAssemblyInfo.targets

```diff
@@ -117,11 +117,11 @@ Copyright (c) .NET Foundation. All rights reserved.
     </ItemGroup>
 
     <ItemGroup Condition="'$(TargetPlatformIdentifier)' != ''
-                         and '$(MinimumOSPlatform)' != ''
```

Comment:
@adamsitnik \ @wli3  do you happen to know what values I should use to make this work for Blazor? What values do I need to specify in the SDK for `TargetPlatformIdentifier` and `MinimumOSPlatform`?